caikit/interfaces/common/data_model/stream_sources.py

```diff
@@ -26,12 +26,12 @@
 
 
 @dataobject(PACKAGE_COMMON)
-class File(DataObjectBase):
+class FileStream(DataObjectBase):
```

Comment:
iiinteresting. I know this is only used by `DataStreamSource`s but the name `FileStream` seems a bit confusing in a vacuum here. Since the goal here is to reference a file that a runtime service can open, should this be named `FileReference`? `LocalFile`? `MountedFile`? `FileMount`?

Since the name of this class ends up as the field name in the api as well, I think it'd be good to make sure that whatever we pick here conveys the meaning well enough to somebody who's just looking at the api spec and trying to make things work

Reply:
I also didn't like `FileStream` all that well. Since this class just contains a string pointing to a file I think `FileReference` makes the most sense. Does that work?